recipes/config.rb

```diff
@@ -40,9 +40,18 @@
 bin = ::File.join(node['qubit_bamboo']['home'], 'bamboo')
 flags = node['qubit_bamboo']['flags'].sort.map { |k, v| " -#{k}=#{v}" }.join ' '
 syslog = node['qubit_bamboo']['syslog'] ? '2>&1 | logger -p user.info -t bamboo' : ''
-bamboo_command = %("#{bin} #{flags} #{syslog}")
+
+node.default['qubit_bamboo']['poise_service']['options'] = {
```

Comment:
Gregoire said that you can move all the block bin/flags/syslog + the poise_service options hash inside the attributes file.
But it's an OPT for me.